src/GUI/GamePanel.java

```diff
@@ -0,0 +1,79 @@
+package GUI;
+
+import Gameplay.*;
+
+import javax.swing.*;
+import java.awt.*;
+import java.io.IOException;
+
+public class GamePanel extends JPanel {
+
+    protected static int XOffset = 100;
+    public static Point deckPosition = new Point(620, 20);
+    public static Point tableauPosition = new Point(20, 150);
+    public static int tableauOffset = 100;
+    private static Deck deck;
+    private static WastePile wastePile;
+    private static Foundations[] foundations;
+    private static Tableau[] tableaus;
+    private ShuffleButton shuffleButton;
+
+    public GamePanel() throws IOException {
+        super.setLayout(null);
+        initializePiles();
+
+        GameMoveListener gameMoveListener = new GameMoveListener();
+        addMouseListener(gameMoveListener);
+        addMouseMotionListener(gameMoveListener);
+    }
+
+
+    private void initializePiles() throws IOException {
+            deck = new Deck(deckPosition.x, deckPosition.y);
```

Comment:
Misaligned indentation for this function

```suggestion
        deck = new Deck(deckPosition.x, deckPosition.y);
```